### DISCUSSION TITLE
Matching the end of line does not seem to be working.

### DISCUSSION THREAD

**Discussion by DasOhmoff** [BurntSushi/ripgrep#1735] [Q&A] [answered]
Hello and thank you for your help.
I have a problem when I try to match the '$'  at the end of lines.

For example if I have a file called `test.txt` in my working directory:
```
-------#
-------)
-------#
```

When I try to find all lines that have a `#` at the end, it prints the last line get printed.
When I try to find all lines that have a `)` at the end, no line gets printed.

![image](https://user-images.githubusercontent.com/16063186/99702877-a8c6e480-2a96-11eb-8e58-e6fdabae5d68.png)

I tried using the `-U` flag, but that does not seem to help.
How can a make it happen so that the `$` character is associated with `end of line`?
By the way, I am working on a `Windows` machine.

**Comment by BurntSushi** [comment, marked as answer]
> By the way, I am working on a `Windows` machine.

You probably need to use the `--crlf` flag. ripgrep's `$` doesn't support detecting Windows newlines.

**Comment by DasOhmoff** [comment]
This is exactly what I wanted to know, thank you!
